doc/man3/X509_check_issued.pod

```diff
@@ -14,23 +14,23 @@ certificate
 
 =head1 DESCRIPTION
 
-This function checks if certificate I<subject> was issued using CA
+X509_check_issued() checks if certificate I<subject> was likely issued using CA
 certificate I<issuer>. This function takes into account not only
-matching of issuer field of I<subject> with subject field of I<issuer>,
-but also compares B<authorityKeyIdentifier> extension of I<subject> with
-B<subjectKeyIdentifier> of I<issuer> if B<authorityKeyIdentifier>
-present in the I<subject> certificate and checks B<keyUsage> field of
-I<issuer>.
+matching of the issuer field of I<subject> with the subject field of I<issuer>,
+but also compares all sub-fields of the B<authorityKeyIdentifier> extension of
+I<subject>, as far as present, with the respective B<subjectKeyIdentifier>,
```

Comment:
s/as far as present/when present/

Reply:
I deliberately chose "as far as present" (two times in this sentence) trying to emphasize that **as many sub-fields as** the AKID of the given subject cert actually has (which may contain any/all of key ID, issuer(s), and serial number), these are compared with the respective fields of the issuer cert (as far as each of these are present).

If using "when present" in my view this does not become clear. If you see a better solution that covers this subtle aspect please let me know.